src/main.rs

```diff
@@ -17,18 +17,19 @@ fn spawn_as_wrapper() {
 }
 
 fn main() {
+    let args: Vec<String> = std::env::args().collect();
+
     #[cfg(not(release))]
-    dbg!(std::env::args());
+    dbg!(&args);
```

Comment:
The  `dbg!` macro gets optimized out, when compiled in `release` mode. Clippy would complain about using the `dbg!` macro here anyway, because this macro should not appear in production code.

Reply:
There is no `not(release)` cfg. The closest is `debug_assertions`.